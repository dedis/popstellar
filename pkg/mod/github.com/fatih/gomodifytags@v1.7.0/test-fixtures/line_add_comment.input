package foo

type foo struct {
	// a comment about b

	bar string
	t   bool

	// a comment about b
	// b []string
}
