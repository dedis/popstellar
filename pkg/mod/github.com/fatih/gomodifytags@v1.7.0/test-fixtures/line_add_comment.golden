package foo

type foo struct {
	// a comment about b

	bar string `json:"bar"`
	t   bool   `json:"t"`

	// a comment about b
	// b []string
}
